proto/viam/app/v1/robot.proto

```diff
@@ -88,6 +88,8 @@ message ProcessConfig {
   string cwd = 4;
   bool one_shot = 5;
   bool log = 6;
+  int32 stop_signal = 7;
+  int64 stop_timeout = 8;
```

Comment:
seems like you should use proto duration for this

Reply:
Good call... I didn't know that was a thing (I knew about timestamp, but didn't realize specifically duration was available.)